web/styles/app_variables.css

```diff
@@ -491,6 +491,9 @@
     --color-background-emoji-picker-emoji-reacted: hsl(195deg 50% 95%);
     --color-border-emoji-picker-emoji-reacted: hsl(195deg 52% 79%);
     --color-background-emoji-picker-emoji-reacted-focus: hsl(195deg 55% 88%);
+
+    /* Rendered markdown table color */
```

Comment:
Two nits here:

1. Let's put these just above the `/* Markdown code colors */` portion of the file
2. Make the heading something more generic, like `/* Markdown colors */`, so that additional variables can be set up there in the future.